packages/flutter/lib/src/gestures/monodrag.dart

```diff
@@ -497,6 +497,13 @@ class HorizontalDragGestureRecognizer extends DragGestureRecognizer {
 
   @override
   bool _isFlingGesture(VelocityEstimate estimate) {
```

Comment:
Couldn't we make DragGestureRecognizer a public method with a default implementation that returns false (introducing another abstract method would break existing subclasses)? So that we wouldn't need any private _isFlingGesture methods?